### PR TITLE
Ft edit get delete articles

### PR DESCRIPTION
**Pull request by goodnessaig1**

### CODE REVIEW

File: server/controller/articleController.js [hunk 1]
```diff
@@ -80,6 +81,109 @@ class ArticleController {
...
+      const { title, article } = req.body;
+      const user = await pool.query(queries.selectArticle,[articleId]);
+
+        if (user.rowCount === 0) return res.status(404).json({ 
+            status: 'Failed',
+            message: 'Article Not Found'
+        });
+        
+        const updatedAt = DateTime.now()
+        const updated = await pool.query(queries.updateArticle,[title, article, updatedAt, articleId]);
```
Comment (courage173) [inline server/controller/articleController.js]: for this you will always be updating the title and article, although its not bad but its always better to do a patch that is only those that are changing. so if only the title is changing update that alone. well it means you have to tweak this a bit. but don't worry lets keep it this way. just wanted to mention this here

File: server/middleware/authorization.js [hunk 2]
```diff
@@ -5,14 +5,16 @@ const authorization = (req, res, next) => {
...
-      next();
-    });
-  } else {
-    return res.status(400).send('Invalid token.');
+   if (!token) return res.status(401).send('Access denied. No token provided.');
+  try {
+    const decodedPayload = jwt.verify(token, 'jwtPrivateKey');
+    req.user = decodedPayload;
+    next();
+  } catch (ex) {
```
Comment (courage173) [inline server/middleware/authorization.js]: what is ex? since you are not using it you can remove it

**Review by courage173 — APPROVED** [review]

File: server/controller/articleController.js (hunk 1 above)
Comment (goodnessaig1) [reply server/controller/articleController.js]: Ok sir

File: server/middleware/authorization.js (hunk 2 above)
Comment (goodnessaig1) [reply server/middleware/authorization.js]: alright sir. I have changed it to err

**Pull request merged**